### DISCUSSION TITLE
restart from named output

### DISCUSSION THREAD

**Discussion by bzdjordje** [epochpic/epoch#168] [Q&A] [answered]
Is it possible to generate a restart dump from a named output block or at least to have an infrequent 'old' output block that still accepts the restart option concurrently with a named block? I am seeing an error for a named output block with restart and cannot have both kinds of blocks.

**Comment by TomGoffrey** [comment, marked as answer]
I'm not sure I understand the question. You can certainly generate restarts from a named output block. I do this often, something like:

```
begin:output
  name = Restart
  file_prefix = restart
  restartable = T
  dt_snapshot = 0.1
end:output
```

Perhaps if you post your input deck, and the error produced, and the version of EPOCH you're using, your question will be clearer.

**Reply by bzdjordje** [reply]
I missed that named output blocks had an additional 'restartable' variable that needed to be set as well, thank you!

**Comment by keithbennett** [comment]
Since this is a support request rather than a potential bug report, it has been moved to the Q&A discussion section.
